fsharp-backend/tests/httpbytestestfiles/simple-request-headers.test

```diff
@@ -0,0 +1,32 @@
+[http-bytes-handler GET /]
+(let relevantHeaders = Dict.filter_v1 request.headers (fun (k, _v) -> String.startsWith_v0 k "header") in
+let body = relevantHeaders |> toString |> String.toBytes_v0 in
+Http.response body 200)
+
+[request]
+GET / HTTP/1.1
+Host: HOST
+Date: Sun, 08 Nov 2020 15:38:01 GMT
+Content-Length: 7
+header1: value1
+Header2: Value2
+Header-3: Value-3
+Header-with-Casing-conflict: test1
+header-WITH-Casing-conflict: test2
+
+ignored
+
+[response]
+HTTP/1.1 200 OK
+Date: xxx, xx xxx xxxx xx:xx:xx xxx
+x-darklang-execution-id: 0123456789
+Server: darklang
+Strict-Transport-Security: max-age=31536000; includeSubDomains; preload
+Content-Length: 114
+
+{
+  header2: "value2",
+  header1: "value1",
+  header-with-casing-conflict: "test1,test2",
+  header-3: "value-3",
+}
```

Comment:
Frustratingly, this test fails. Visually, the expected and actual response bodies match, but there are subtle byte-level things apparently preventing this from failing
![image](https://user-images.githubusercontent.com/906686/183928933-fc10171c-5916-473b-9014-a2bdbf4f761e.png)

I'm not sure how to deal with this; advice/assist would be appreciated.

Reply:
I'm really not sure. Maybe debug in real time?

Reply:
https://github.com/darklang/dark/pull/4353/commits/39fceabd7b55e9b161a13928289a67d4df5c52e0#diff-41aa10236dd071016d57d4da800039add6db30ca7f1450964f22da98e80b1cadR27

VS Code was trimming a space character annoyingly. This isn't the most elegant solution, but it works.